src/covid19model/visualization/optimization.py

```diff
@@ -0,0 +1,146 @@
+import datetime
+import random
+import matplotlib
+import matplotlib.pyplot as plt
+import matplotlib.dates as mdates
+import pandas as pd
+import numpy as np
+
+
+# From Color Universal Design (CUD): https://jfly.uni-koeln.de/color/
+colorscale_okabe_ito = {"orange" : "#E69F00", "light_blue" : "#56B4E9",
+                        "green" : "#009E73", "yellow" : "#F0E442",
+                        "blue" : "#0072B2", "red" : "#D55E00",
+                        "pink" : "#CC79A7", "black" : "#000000"}
```

Comment:
The colorscale is already defined in another file, output. I would not copy-paste these in each file, as it is important we keep the code as DRY (Don't Repeat Yourself) as possible in order to keep it maintainable. So either load it from output.py:

```suggestion
from .output import colorscale_okabe_ito
```
or we should make a more common file for the general functionalities.

Reply:
@stijnvanhoey  Can you give me some pointers on an appropriate file name and location? I'll make this modification.

Reply:
I would add the `plt.rcParams.update` in the __init__.py file of this vis folder (see other comment) and I would add the `colorscale_okabe_ito` and `_apply_tick_locator` to a new file, eg. `utils.py`?